pkg/agent/cniserver/interface_configuration_linux.go

```diff
@@ -241,24 +241,29 @@ func (ic *ifConfigurator) configureContainerLinkVeth(
 	containerIfaceName string,
 	mtu int,
 	result *current.Result,
+	ifaceMAC net.HardwareAddr,
```

Comment:
nit: I think the original `mac` was fine as the parameter name. It's more consistent with the existing `mtu` parameter